site/de/publications.md

```diff
@@ -8,103 +8,113 @@ lang-ref: publications
 ## Publikationen
```

Comment:
those are the presentations, not the publications. publications should stay on top of the document though

Reply:
fixed in 3afafc85